lib/validation/rules/requiredYmd.ts

```diff
@@ -1,14 +1,18 @@
 import { helpers } from '@vuelidate/validators'
 import { type Ymd } from '../../support/Day'
+import { createRule } from '../Rule'
 import { requiredYmd as baseRequiredYmd } from '../validators/requiredYmd'
 
 type YmdType = 'y' | 'm' | 'd'
 
+export const message = {
+  en: 'The field is required.',
+  ja: 'この項目は必須です。'
+}
+
 export function requiredYmd(required?: YmdType[], msg?: string) {
-  return helpers.withMessage(
-    () => msg ?? 'The field is required.',
-    (value: Ymd) => {
-      return !helpers.req(value) || baseRequiredYmd(value, required)
-    }
-  )
+  return createRule({
+    message: ({ lang }) => msg ?? message[lang],
+    validation: (value: Ymd) => !helpers.req(value) || baseRequiredYmd(value, required)
```

Comment:
`!helpers.req(value)` may not be needed according to `createRule` implementation, but not so sure.

Reply:
Fixed: 5c20e70